src/twin/system_info.rs

```diff
@@ -31,6 +31,85 @@ lazy_static! {
     };
 }
 
+pub enum RootPartition {
+    A,
+    B,
+}
+
+impl RootPartition {
```

Comment:
probably should be it's own file. it is used for system_info and firmware_update

Reply:
created a task